src/node/ext/call.cc

```diff
@@ -521,6 +523,7 @@ Call::Call(grpc_call *call) : wrapped_call(call),
 Call::~Call() {
   if (wrapped_call != NULL) {
```

Comment:
Since that code is duplicated into line 571, I'm usually more in favor of adding a "destroy" method that's containing the code in a unique manner, but called from the two locations. That way you can keep the `if (wrapped_call != NULL)` logic segregated into that specialized function.